data_structures/stacks/stack.py

```diff
@@ -66,17 +66,18 @@ def peek(self) -> T:
 
     def is_empty(self) -> bool:
         """Check if a stack is empty."""
-        return not bool(self.stack)
+        return (not self.stack)
 
     def is_full(self) -> bool:
+        """Check if a stack is full."""
         return self.size() == self.limit
 
     def size(self) -> int:
         """Return the size of the stack."""
         return len(self.stack)
 
     def __contains__(self, item: T) -> bool:
```

Comment:
Please sort all the dunder `__functions__()` before the normal functions.

Reply:
Meh